index.js

```diff
@@ -71,24 +72,35 @@ module.exports = function(options) {
     }));
 
     trees.push(compileTS('tsconfig.tests.json', projectPath, tsinclude));
-
   } else {
-    let es2015 = compileTS('tsconfig.json', projectPath, tsinclude);
-    trees.push(es2015);
-
-    let es5 = toES5(es2015, {
-      sourceMap: 'inline'
-    });
-
-    let namedAMD = toNamedAMD(es5);
-    trees.push(namedAMD);
-
-    let amd = concat(namedAMD, {
-      inputFiles: ['**/*.js'],
-      outputFile: 'amd/' + projectName + '.js'
-    });
-
-    trees.push(amd);
+    let es2017Modules = compileTS('tsconfig.json', projectPath, tsinclude);
+    let filteredEs2017Modules = funnel(es2017Modules, { include: ['**/*.js'] });
+    let es5Modules = toES5(filteredEs2017Modules, { sourceMap: 'inline' });
+    let namespacedEs5Modules = funnel(es5Modules, { destDir: projectName });
+    let es5Amd = toNamedAmd(namespacedEs5Modules);
+    let es2017CommonJs = toNamedCommonJs(filteredEs2017Modules);
+    let es5CommonJs = toNamedCommonJs(es5Modules);
+
+    trees.push(funnel(filteredEs2017Modules, {
+      destDir: 'modules/es2017',
+      annotation: 'modules-es2017'
+    }));
+    trees.push(funnel(es5Modules, {
+      destDir: 'modules/es5',
+      annotation: 'modules-es5'
+    }));
+    trees.push(funnel(es5Amd, {
```

Comment:
I think we need to add `srcDir: projectName,` here as well to prevent the nested dirs from appearing in `amd/es5`